torch/fx/passes/_tensorify_python_scalars.py

```diff
@@ -203,7 +203,7 @@ def _sympy_interp(expr: sympy.Expr) -> MetaProxy:
                 and node.target is torch.ops.aten._local_scalar_dense.default
             ):
                 dtype = node.args[0].meta["val"].dtype
-                if dtype != torch.float64:
+                if dtype != torch.float64 and dtype != torch.float32:
```

Comment:
shall we find a way to handle all float types and add tests for when input is float16 ..etc